tests/puppeteer/pages/BO/customers/index.js

```diff
@@ -276,4 +295,22 @@ module.exports = class Customers extends BOBasePage {
     await this.clickAndWaitForNavigation(enable ? this.bulkActionsEnableButton : this.bulkActionsDisableButton);
     return this.getTextContent(this.alertSuccessBlockParagraph);
   }
+
+  /* Sort functions */
+  /**
+   * Sort table by clicking on column name
+   * @param sortBy, column to sort with
+   * @param sortDirection, asc or desc
+   * @return {Promise<void>}
+   */
+  async sortTable(sortBy, sortDirection = 'asc') {
```

Comment:
```suggestion
  async sortTable(sortBy, sortDirection {
```
You always define sorting direction.

Reply:
I deleted for now, but maybe we will need it later